src/fsharp/NicePrint.fs

```diff
@@ -1405,12 +1405,22 @@ module InfoMemberPrinting =
             | None -> tagProperty
             | Some vref -> tagProperty >> mkNav vref.DefinitionRange
         let nameL = DemangleOperatorNameAsLayout tagProp pinfo.PropertyName
+        let accessibilityText =
+            if pinfo.HasGetter && not pinfo.HasSetter then
+                "get"
+            elif pinfo.HasSetter && not pinfo.HasGetter then
+                "set"
+            else
+                "get, set"
+
         wordL (tagText (FSComp.SR.typeInfoProperty())) ^^
         layoutTyconRef denv pinfo.ApparentEnclosingTyconRef ^^
         SepL.dot ^^
         nameL ^^
         RightL.colon ^^
-        layoutType denv rty
+        layoutType denv rty ^^
+        wordL (tagKeyword "with") ^^
```

Comment:
`"with"` will need to be localized.

Reply:
Is that true given that it's a keyword?

Reply:
Good point, I withdraw my change request, then.